modules/app/src/main/resources/assets/icons/icons.less

```diff
@@ -1,170 +1,9 @@
 @font-face {
   font-family: 'icomoon-studio-app';
   src:
-    url('fonts/icomoon-studio-app.woff2') format('woff2'),
```

Comment:
This is wrong. WOFF2 should be the primary format used, WOFF should be fallback used for compatibility with older browsers. Restore `icomoon-studio-app.woff2`, make sure it has the the same contents as `icomoon-studio-app.woff`. Also add `icomoon-studio-lib.woff2` and make sure it's used as primary format in `@font-face`